js/pieChart.js

```diff
@@ -1,254 +1,323 @@
-        vis.multiLevelData[0].forEach((levelData, i) => {
-            let currentPieWidth = (i + 1) * pieWidth;
-
-            arc.outerRadius(currentPieWidth - 1)
-                .innerRadius(i * pieWidth);
-
-             // Select all arc groups and bind new data
-            let g = vis.svg.selectAll(".arc" + i)
-                .data(pie(levelData.subData));
-
-            // Exit phase for old elements
-            g.exit().transition().duration(1000)
-                .style("opacity", 0)
-                .remove();
-
-            // Enter phase for new elements
-            let enterG = g.enter().append("g")
-                .attr("class", "arc" + i);
-
-            enterG.append("path")
-                .attr("d", arc)
-                .style("fill", d => vis.color(d.data.brand))
-                .on("mouseover", function(event, d) {
-                    tooltip.transition().duration(200).style("opacity", 1);
-                    tooltip.html(`
+      enterG
+        .append("text")
+        .attr("transform", (d) => "translate(" + arc.centroid(d) + ")")
+        .attr("dy", ".35em")
+        .style("text-anchor", "middle")
+        .style("font-size", i === 0 ? "10px" : "14px")
+        .text((d) => d.data.percentageWaste.toFixed(1) + "%")
+        .on("mouseover", function (event, d) {
+          tooltip.transition().duration(200).style("opacity", 1);
+          tooltip
+            .html(
+              `
                         <strong>Brand:</strong> ${d.data.brand} <br>
                         <strong>Year:</strong> ${d.data.year} <br>
                         <strong>Waste Generation:</strong> ${d.data.waste.toLocaleString()}
-                    `)
-                    .style("left", (event.pageX + 10) + "px")
-                    .style("top", (event.pageY - 20) + "px");
-                })
-                .on("mousemove", function(event) {
-                    tooltip.style("left", (event.pageX + 10) + "px")
-                        .style("top", (event.pageY - 20) + "px");
-                })
-                .on("mouseout", function() {
-                    tooltip.transition().duration(200).style("opacity", 0);
-                });
-
-            enterG.append("text")
-                .attr("transform", d => "translate(" + arc.centroid(d) + ")")
-                .attr("dy", ".35em")
-                .style("text-anchor", "middle")
-                .style("font-size", (i === 0) ? "10px" : "14px")
-                .text(d => d.data.percentageWaste.toFixed(1) + "%");
-
-            g.merge(enterG).transition().duration(1000)
-                .style("opacity", 1) 
-                .select("path")
-                .attr("d", arc)
-                .style("fill", d => vis.color(d.data.brand))
-                .attr("transform", "scale(1.05)") 
-                .transition().duration(500)
-                .attr("transform", "scale(1)"); 
-
-            g.merge(enterG).transition().duration(1000)
-                .select("text")
```

Comment:
you can safely ignore all these changes, i just copy/pasted Farah's file without changing anything. 

I accidentally deleted it on  my local when working 😓